chat-plugins/datasearch.js

```diff
@@ -434,7 +434,7 @@ function runDexsearch(target, cmd, canAll, message) {
 
 			let typeIndex = target.indexOf('type');
```

Comment:
Why isn't this line replaced with a simple detection of `target.endsWith('type')`?